### PR TITLE
Add GitHub actions and fix #28

### LINKED ISSUE #28: Missing homepage and issues tags inside the fabric.mod.json file.
Please don't be mad for reporting something so pointless.

### PR DESCRIPTION
**Pull request by Genau6502**
- GitHub actions on push and PR opened
- Fixed #28 by adding the correct values in ```fabric.mod.json```

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    strategy:
+      matrix:
+        # Use these Java versions
+        java: [
+          1.8,  # Minimum supported by Minecraft
+          11,   # Current Java LTS
+          15    # Latest version
+        ]
+        # and run on both Linux and Windows
+        os: [ubuntu-20.04, windows-latest]
```
Comment (katekerllenevich) [inline .github/workflows/main.yml:15]: running on windows has no effect and iirc github charges money for it.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: make gradle wrapper executable
+        if: ${{ runner.os != 'Windows' }}
+        run: chmod +x ./gradlew
+      - name: build
+        run: ./gradlew build
+      - name: capture build artifacts
+        if: ${{ runner.os == 'Linux' && matrix.java == '11' }}
```
Comment (katekerllenevich) [inline .github/workflows/main.yml:32]: all the artifacts should be uploaded

**Review by katekerllenevich — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (Genau6502) [reply .github/workflows/main.yml:15]: spottedleaf has pro, which gets you 2000 minutes of free actions

Comment (Aurelien30000) [reply .github/workflows/main.yml:15]: image
😅

File: .github/workflows/main.yml (hunk 2 above)
Comment (Genau6502) [reply .github/workflows/main.yml:32]: They shouldn't, as long as they all build there isn't a problem

File: .github/workflows/main.yml (hunk 1 above)
Comment (katekerllenevich) [reply .github/workflows/main.yml:15]: building on windows is still unnecessary

File: .github/workflows/main.yml (hunk 2 above)
Comment (katekerllenevich) [reply .github/workflows/main.yml:32]: it justs complicates things. there is no reason. this also supports my windows idea. if you aren't doing anything with windows jars then why build them.

File: .github/workflows/main.yml (hunk 1 above)
Comment (ImperatorStorm) [reply .github/workflows/main.yml:15]: I agree  with not OM3GA, there is no reason to compile on windows and linux when dealing with java, its cross-platform

**Comment by Genau6502** [comment]
Since clearly none of you want it I'm going to close this

**Pull request closed without merging**